template/bash_profile.template.bash

```diff
@@ -32,6 +32,10 @@ export TODO="t"
 
 # Set this to false to turn off version control status checking within the prompt for all themes
 export SCM_CHECK=true
+# Set to actual location of gitstatusd if installed
```

Comment:
Same here: It's not pointing to gitstatusd.

Reply:
done